src/media/css/feed/brands.styl

```diff
@@ -67,7 +67,27 @@
 }
 
 .brand-landing-header-desktop {
+    margin-top: 22px;  // To get 40px. Combine with 18px #page padding.
     display: none;
+    position: relative;
+    z-index: 1;
+
+    h1 {
+        desktop_mega();
+        margin: 20px 0 40px;
+        text-align: center;
+    }
+}
+
+.brand-landing-curve {
+    background: rgb(250, 250, 250);
+    border-radius: 50% 50% 0 0;
+    height: 145px;
+    position: absolute;
+    bottom: -40px;
+    width: 120%;
+    left: -10%;
```

Comment:
Percentages can give messy results on mobile but this should be all desktop anyway.